README.md

```diff
@@ -5,13 +5,13 @@ For example in the gateway's `sendData()` methods we can do:
 
     ```PHP
     $event_dispatcher = $this->httpClient->getEventDispatcher();
-    $event_dispatcher->dispatch(Constants::OMNIPAY_REQUEST_BEFORE_SEND, new RequestEvent($request));
+    $event_dispatcher->dispatch(Constants::OMNIPAY_REQUEST_BEFORE_SEND, new RequestEvent($request, 'auth'));
```

Comment:
It'd be helpful to clarify that `'auth'` is the name of the request. Right now, it looks like maybe you're supposed to say auth everywhere, especially since you're saying to do this in sendData and most requests use the same sendDate function. What about something like `get_class($this)` or `$this->getRequestName()`? (getRequestName isn't actually a standard thing, but they could add it...)

Reply:
I doesn't really need to be a function right? I can just do `$request_name // The name of the API endpoint being called`

Reply:
yeah that's fine

Reply:
Cool, done!